client-core/Cargo.toml

```diff
@@ -53,7 +53,7 @@ ripemd160 = "0.8.0"
 test-common = { path = "../test-common" }
 
 [features]
-default = ["sled", "sgx-obfuscation"]
+default = ["sled", "edp-obfuscation"]
```

Comment:
It seems to me, no need to keep features `edp-obfuscation` and `sgx-obfuscation`? only need one of them, and they can be build in normal environment, right?

Reply:
Eventually, we'll depricate `sgx-obfuscation` after `edp-obfuscation` is sufficiently tested.